### ISSUE TITLE
plugins/tiff_stream.h: error: too few arguments to function call, expected 5, have 4

### ISSUE THREAD

**Issue by thierry-FreeBSD** [dtschump/CImg#335]
Trying to compile the examples of v 3.0.0 (with clang) produces this error:
```
In file included from use_tiff_stream.cpp:52:
In file included from ../CImg.h:11779:
../plugins/tiff_stream.h:162:35: error: too few arguments to function call, expected 5, have 4
        frame._load_tiff(tif,l,0,0);
        ~~~~~~~~~~~~~~~~          ^
use_tiff_stream.cpp:68:9: note: in instantiation of member function 'cimg_library::CImg<unsigned short>::load_tiff' requested here
  imgIn.load_tiff(&inFile);
        ^
../CImg.h:54062:14: note: '_load_tiff' declared here
    CImg<T>& _load_tiff(TIFF *const tif, const unsigned int directory, unsigned int *const bits_per_value,
             ^
117 warnings and 1 error generated.
```

**Comment by thierry-FreeBSD** [comment]
Same error with v 3.0.1.

**Comment by dtschump** [comment]
You are right. Just forgot to fix this one, sorry.
Now fixed, and released as a new CImg 3.0.1 package.

**Comment by thierry-FreeBSD** [comment]
Great, thanks!
